charts/netdata/templates/clusterrole.yaml

```diff
@@ -29,6 +29,13 @@ rules:
       - "get"
       - "list"
       - "watch"
+  - apiGroups: ["batch"]
+    resources:
+      - "cronjobs"
```

Comment:
I would probably also add jobs while we are at it.

Reply:
added